sentry_v2/README.md

```diff
@@ -0,0 +1,33 @@
+# Sentry V2 Sample
+
+This sample shows how to configure [Sentry](https://sentry.io) SDK V2 to intercept and capture errors from the Temporal SDK.
+
+Note: This is a small modification of the original example [sentry (v1)](../sentry) which seems to have
+issues in the Workflow interceptor, as there are modules (`warnings` and `threading`) that Sentry uses that aren't passed through
+to Temporal's sandbox properly, causing the Workflows to fail to execute.
```

Comment:
If Sentry V1 doesn't work anymore, let's replace the sample instead of adding a new one

Reply:
Hey @cretz - I haven't been able to test this, but I suspect now the example didn't work for me because when I installed sentry-sdk, it will have defaulted to the latest version (V2), but the example only works with V1. I imagine if you pin the version to the one in the example, it might work.

It does look like development on the V1 SDK has stopped now (last feature release was [in April](https://github.com/getsentry/sentry-python/releases/tag/1.45.0) with a security patch [in July](https://github.com/getsentry/sentry-python/releases/tag/1.45.1)). 

So, I will let you decide if we should replace the V1 example. I'll update the PR as necessary.

Reply:
Hrmm, assuming the current `sentry` works, how about we just move the current one to `sentry_v1` and make yours just `sentry`? Then we can come back and delete `sentry_v1` later when they officially EOL it.

Reply:
@cretz I renamed sentry -> sentry_v1 and kept sentry_v2 as is, so the commit history doesn't get skewed for the original impl. Let me know if you'd prefer to make sentry_v2 -> sentry.

Reply:
> Let me know if you'd prefer to make sentry_v2 -> sentry.

That does seem to make more sense, but I don't know Sentry's stance on supporting v1. If we support v1 and v2, I think we should have different groups.

Reply:
Does Sentry still support v1? If it is EOL, maybe we should consider it EOL too.